### PR TITLE
refactor,chore(egen) : refactored code as per template guidelines

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes
1.Adds Colab authentication and kernal restart steps.
2.Adds colab enterprise logo with link.
3.Removes IS_TESTING condition.
4.Replace REGION with LOCATION.
5.Removed UUID code
6.Removed unused modules

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb [hunk 1]
```diff
@@ -515,18 +462,18 @@
...
         "\n",
-        "- `dataset`: The `Dataset` resource to train the model.\n",
-        "- `model_display_name`: The human readable name for the trained model.\n",
-        "- `training_fraction_split`: The percentage of the dataset to use for training.\n",
-        "- `test_fraction_split`: The percentage of the dataset to use for test (holdout data).\n",
-        "- `validation_fraction_split`: The percentage of the dataset to use for validation.\n",
-        "- `target_column`: The name of the column to train as the label.\n",
-        "- `budget_milli_node_hours`: (optional) Maximum training time specified in unit of millihours (1000 = hour).\n",
-        "- `disable_early_stopping`: If `True`, training maybe completed before using the entire budget if the service believes it cannot further improve on the model objective measurements.\n",
+        "- dataset: The dataset resource to train the model.\n",
```
Comment (kittyabs) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:465]: put the back ticks back in:
-  `dataset`: The dataset resource to train the model.\n"
And for all the others below, down to row 72

File: notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb [hunk 2]
```diff
@@ -465,24 +412,24 @@
...
+        "  - regression: A tabular regression model.\n",
+        "- column_transformations: (Optional): Transformations to apply to the input columns\n",
+        "- optimization_objective: The optimization objective to minimize or maximize.\n",
         "  - binary classification:\n",
-        "    - `minimize-log-loss`\n",
-        "    - `maximize-au-roc`\n",
-        "    - `maximize-au-prc`\n",
-        "    - `maximize-precision-at-recall`\n",
-        "    - `maximize-recall-at-precision`\n",
+        "    - minimize-log-loss\n",
```
Comment (kittyabs) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:422]: put the back ticks back in all the way through line 432

File: notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb [hunk 3]
```diff
@@ -465,24 +412,24 @@
         "\n",
         "An AutoML training pipeline is created with the `AutoMLTabularTrainingJob` class, with the following parameters:\n",
         "\n",
-        "- `display_name`: The human readable name for the `TrainingJob` resource.\n",
-        "- `optimization_prediction_type`: The type task to train the model for.\n",
-        "  - `classification`: A tabuar classification model.\n",
-        "  - `regression`: A tabular regression model.\n",
-        "- `column_transformations`: (Optional): Transformations to apply to the input columns\n",
-        "- `optimization_objective`: The optimization objective to minimize or maximize.\n",
+        "- display_name: The human readable name for the `TrainingJob` resource.\n",
```
Comment (kittyabs) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:415]: put the back ticks back in all the way down to row 420

File: notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb [hunk 4]
```diff
@@ -617,7 +564,7 @@
       "source": [
         "### Make test item\n",
         "\n",
-        "You will use synthetic data as a test data item. Don't be concerned that we are using synthetic data -- we just want to demonstrate how to make a prediction."
+        "You use synthetic data as a test data item. Don't be concerned that you are using synthetic data -- you just want to demonstrate how to make a prediction."
```
Comment (kittyabs) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:567]: contract "you are" to "you're"

**Review by kittyabs — COMMENTED** [review]
Please put the back ticks back in where indicated. And other edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**